src/requests/getForecast.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-console */
+import axios from "axios";
+
+const getForecast = async (
+  setLocation,
+  setForecasts,
+  setSelectedDate,
+  setErrMessage,
+  location,
+  units
+) => {
+  const endpoint = `https://api.openweathermap.org/data/2.5/forecast?q=${location}&units=${units}&APPID=d1cad75804f6bd996f5d83905ac66876`;
```

Comment:
Does this contain any values that you dont want to be made public? They should be in an `env` file if so